Content.Server/Flash/FlashSystem.cs

```diff
@@ -201,4 +256,20 @@ public FlashAttemptEvent(EntityUid target, EntityUid? user, EntityUid? used)
             Used = used;
         }
     }
+
+    public sealed class AfterFlashedEvent
+    {
+        public readonly EntityUid Target;
+        public readonly EntityUid? User;
+        public readonly EntityUid? Used;
+
+        public AfterFlashedEvent(EntityUid target, EntityUid? used, EntityUid? user)
+        {
+            Target = target;
+            Used = used;
+            User = user;
+        }
+    }
+
+
```

Comment:
Make it a readonly struct event with ByRef attribute and needs documenting on when it's raised.

Reply:
I did the readonly struct but I'm not sure how to add the ByRef.